SQF/dayz_code/stringtable.xml

```diff
@@ -2406,12 +2414,8 @@
 			<German>Ziegelstein</German>
 		</Key>
 		<Key ID="STR_ITEM_DESC_equip_brick">
-			<English>A brick used in crafting.</English>
```

Comment:
If anyone is wondering why these translations were removed, it's because this string wasn't used anywhere.